### PR TITLE
Dash bounding box image annotation

### PR DESCRIPTION
**Pull request by nicholas-esterer**
# App pull request
- This is a new app

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/bounding-box/

## Workflow

- I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.

## The pre-review review

I have addressed all of the following questions:

- [ ] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.) yes 
- [ ] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)] yes
- [ ] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.) no

### CODE REVIEW

**Comment by emmanuelle** [comment]
The structure of the repo is that each app is a directory inside apps, like here your `dash-image-annotation` and that the app file is directly in this directory, and not in a subdirectory like here. This is required for the automatic deployment of apps (you can take a look at the `predeploy.py` script at the top of the repo). Please use `git mv` to move the files one level up.

File: apps/dash-image-annotation/dash-bounding-box-image-annotation/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## About this app
+
+This shows how Dash can be used to make an application that collects annotations
+of the content in parts of images. These annotations could be downloaded and
+used to train an image content classifier.
+
+## How to run this app locally
+
+(The following instructions are for unix-like systems)
```
Comment (emmanuelle) [inline apps/dash-image-annotation/dash-bounding-box-image-annotation/README.md:11]: why? On Windows Anaconda for example provides a unix-like shell which can be used to execute the instructions below.

File: apps/dash-image-annotation/dash-bounding-box-image-annotation/utils.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+def indices_to_path(indices):
+    """From numpy array to SVG path
+    """
+    path = "M"
+    for row in indices.astype(str):
+        path += row[0] + "," + row[1] + "L"
+    path = path[:-1] + "Z"
+    return path
```
Comment (emmanuelle) [inline apps/dash-image-annotation/dash-bounding-box-image-annotation/utils.py:70]:
```suggestion
```

Comment (emmanuelle) [reply apps/dash-image-annotation/dash-bounding-box-image-annotation/utils.py:70]: this code is used only for the bounding box app I think so let's remove it if it's unused here.

File: apps/dash-image-annotation/dash-bounding-box-image-annotation/app.py [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+                dcc.Dropdown(
+                    id="annotation-type-dropdown",
+                    options=[{"label": t, "value": t} for t in annotation_types],
+                    value=DEFAULT_ATYPE,
+                    clearable=False,
+                ),
+                html.H6("Choose image"),
+                html.Button("Previous", id="previous", className="button"),
+                html.Button("Next", id="next", className="button"),
+                html.Button("Add Shape", id="add-shape", className="button"),
```
Comment (emmanuelle) [inline apps/dash-image-annotation/dash-bounding-box-image-annotation/app.py:267]: do we still need this "add shape" button?

File: apps/dash-image-annotation/dash-bounding-box-image-annotation/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Dash Bounding Box Image Annotation
+
+## About this app
+
+This shows how Dash can be used to make an application that collects annotations
+of the content in parts of images. These annotations could be downloaded and
```
Comment (emmanuelle) [inline apps/dash-image-annotation/dash-bounding-box-image-annotation/README.md:6]:
```suggestion
of the content in parts of images. These bounding-box annotations could be downloaded and
```

File: apps/dash-image-annotation/dash-bounding-box-image-annotation/app.py (hunk 3 above)
Comment (nicholas-esterer) [reply apps/dash-image-annotation/dash-bounding-box-image-annotation/app.py]: Not really :upside_down_face:

**Comment by emmanuelle** [comment]
(you'll need to merge master first before being able to merge, I think)

**Pull request merged**